lab_1/main.py

```diff
@@ -1,94 +1,121 @@
-"""
-Lab 1
-A concordance extraction
-"""
+import math
 
 
+def checker(con_object, type_ob):
+    if type_ob is list and type(con_object) is list:
+        len_ = len(con_object) > 0
+        bull_ = None not in con_object and bool not in con_object
+
+        if len_ and bull_:
+            return True
+    elif type_ob is str and type(con_object) is str and len(con_object) >= 0:
+            return True
+    elif type_ob is int and type(con_object) is int and con_object >= 0:
+        return True
+
+    return False
+
+
+# токенезирование текста - пунктуация и заглавные
 def tokenize(text: str) -> list:
-    """
-    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
-    :param text: the initial text
-    :return: a list of lowercased tokens without punctuation
-    e.g. text = 'The weather is sunny, the man is happy.'
-    --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
-    """
-    pass
+    tokens = []
 
+    for token in text.lower().split():
+        token = "".join([a for a in token if a.isalpha()])
+        tokens.append(token)
+    return tokens
 
+
+# получение стоп-слов
+def get_stop_words() -> list:
+    with open("stop_words.txt") as file:
```

Comment:
`open` takes some other arguments like encoding and mode - it is important to pass them so that everyone opens it the same way even on the different operating systems.